.github/workflows/testworkflow.yml

```diff
@@ -0,0 +1,60 @@
+name: testworkflow
+
+on: # run any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - main
+      - 'releases/*'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+
+  testworkflow:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - run: echo 'https://github.com/fair-software/howfairis' > urls.txt
+      - uses: tortellini-tools/action@main
+        with:
+          repositories: urls.txt
+
+      - run: ls -l .tortellini/out/fair-software/howfairis/gl-license-scanning-report.json
+     
+      - run: |
+          git clone --depth 1 https://github.com/oss-review-toolkit/ort.git /tmp/ort
+          DOCKER_BUILDKIT=1 docker build -t ort /tmp/ort
+          mkdir -p conf
+          curl -L https://github.com/oss-review-toolkit/ort/raw/master/examples/rules.kts > conf/rules.kts
+          curl -L https://github.com/oss-review-toolkit/ort/raw/master/examples/license-classifications.yml > conf/license-classifications.yml
+          curl -L https://github.com/oss-review-toolkit/ort/raw/master/examples/curations.yml > conf/curations.yml
+          git clone https://github.com/fair-software/howfairis
```

Comment:
I would prefer to make it more similar to the dir tree used by tortellini, for example, I'd clone into in/fair-software/howfairis but let's ask the others see what they think/.

Reply:
Ok, the directory `.ort` is now used for storing all `ort` files, for example in `.ort/in/fair-software/howfairis` and `.ort/out`